tests/cases/fourslash/noIncorrectParamaterCompletions.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path='fourslash.ts' />
+
+//// f(a = /*1*/, b) { }
+//// f(a = a/*2*/, b) { }
+//// f(a = a + /*3*/, b) { }
```

Comment:
These aren't valid declarations of methods or functions.

Reply:
Also, it would be good to test out that within `b`, you still do get completions for `a`.

Reply:
Oops was too focused on ide testing, now fixed